src/store/index.js

```diff
@@ -38,6 +38,11 @@ export const state = {
 export function handleWidgetData(data) {
   state.widgetData = data
 
+  if (/:/.test(data.widgetPackage)) {
+    state.widgetData.widgetLayout = Fliplet.Widget.getAttributes(data.widgetId)
+    state.widgetData.widgetPackage = data.widgetPackage.split(':')[0]
```

Comment:
The package won't have `:`. this will just be the normal package name.

Reply:
@YaroslavOvdii We won't be passing the `:layoutName` anymore via query. That's why you get the layout via `Fliplet.Widget.getAttributes(data.widgetId)`.

In my instructions I said: `Save the package only as currently is`

Reply:
This means that you don't need to reassign `state.widgetData.widgetPackage`. Also means that `Fliplet.Widget.getAttributes(data.widgetId)` will return a value or nothing. @squallstar needs to confirm what will return if a component doesn't have an attribute to get.

Reply:
Most likely we will return `undefined` when the attribute is not found.

Reply:
@hcarneiro Fixed